packages/ubuntu_provision/lib/src/eula/eula_page.dart

```diff
@@ -0,0 +1,98 @@
+import 'dart:io';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:pdfrx/pdfrx.dart';
+import 'package:ubuntu_provision/interfaces.dart';
+import 'package:ubuntu_provision/providers.dart';
+import 'package:ubuntu_provision/src/eula/eula_l10n.dart';
+import 'package:ubuntu_utils/ubuntu_utils.dart';
+import 'package:ubuntu_wizard/ubuntu_wizard.dart';
+import 'package:yaru/yaru.dart';
+
+class EULAPage extends ConsumerStatefulWidget with ProvisioningPage {
+  const EULAPage({super.key});
+
+  @override
+  ConsumerState<EULAPage> createState() => _EULAPageState();
+}
+
+class _EULAPageState extends ConsumerState<EULAPage> {
+  bool _hasAcceptedTerms = false;
+
+  @override
+  Widget build(BuildContext context) {
+    final localeModel = ref.read(localeModelProvider);
+    final lang = EULALocalizations.of(context);
+    final eulaFile = File(
+        '/usr/share/desktop-provision/eula/EULA_${localeModel.selectedLocale?.languageCode}.pdf');
+
+    return WizardPage(
+      title: YaruWindowTitleBar(
+        title: Text(lang.eulaPageTitle),
+      ),
+      content: Center(
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Text(lang.eulaReviewTerms,
+                style: Theme.of(context).textTheme.titleLarge),
+            Text(lang.eulaReadAndAcceptTerms),
+            const SizedBox(height: kWizardSpacing),
+            Expanded(
+              child: PdfViewer.file(
+                eulaFile.existsSync()
+                    ? eulaFile.path
+                    : '/usr/share/desktop-provision/eula/EULA.pdf',
```

Comment:
We use the fallback one situated at `/usr/share/desktop-provision/eula/EULA.pdf`
Otherwise, there is an error displayed directly in the `PDFViewer` widget.
We could decide not to display the PdfViewer if none of the file exist, and replace it by a custom error string.

Reply:
It was the fallback I meant, but I saw that it's defined in the customization guide. I think it's fine displaying the error in the `PdfViewer`, it's hopefully only something that the OEM folks should see if they set it up wrong.